src/server/src/main/resources/db/cassandra/013_multithreaded_repair.cql

```diff
@@ -0,0 +1,4 @@
+--
+-- Upgrade to allow segments to contain several token ranges
+
+ALTER TABLE repair_unit_v1 ADD repair_thread_count int;
```

Comment:
Even on Cassandra-2.2 (CQL-3.3.1) the `tinyint` data type was available.
As I don't think we're going to say thread count 128+ maybe tinyint is the appropriate type to use here.

Reply:
The database is already very small and the repair unit table is one with just a few rows in it, so I don't feel a strong need to use `tinyint` even if the value is planned to be very small (limited to 4 actually).

Reply:
Sure, but given the effort to make that change why not be more accurate? It's not just about the disk space used, but the intuitiveness that the correct type implies in its schema.